gsoc_summary.md

```diff
@@ -0,0 +1,82 @@
+# GSOC 2020 Work product submission.
+
+## Summary of GSoC project with conda-forge (NumFocus)
+
+### Introduction
+  Throughout this Google Summer of code ([GSoC](https://summerofcode.withgoogle.com/)) project I, along with my mentors, aimed to improve the conda-forge "auto-tick, by splitting  bot versions update cron job from the main environment, more specifically, separating the version updates structure to its own microservice. Which in return, improved to a more organic bot ecosystem, not only lessening the burden to the original main process, but also granting the bot team a more versatile way to track the packages version datas, and consecutively it's respective issues.
```

Comment:
datum, singular, data plural. Nowadays one can use data as singulars but datas does not exist (yet!).

Reply:
thanks for the advice 😄